pyproject.toml

```diff
@@ -70,7 +70,7 @@ document = [
   "matplotlib!=3.6.0",
   "pandas",
   "pillow",
-  "plotly>=4.9.0",  # optuna/visualization.
+  "plotly>=4.9.0,<6.3.0",  # optuna/visualization. TODO(nabe): <6.3.0 is a hotfix, so please remove it once the bug is fixed.
```

Comment:
Could you specify the upper version constraint on `windows-tests.yml`? I think this constraint should not be affected to all platforms.
https://github.com/optuna/optuna/blob/e6b2e42d0d1634700eb2688f5afee865253d6c94/.github/workflows/windows-tests.yml#L55-L58